settings.gradle

```diff
@@ -41,3 +37,8 @@ boolean omnitureDependenciesAreSatisfied() {
     File dep = file("${System.getProperty("user.home")}/libs/adobeMobileLibrary.jar")
     return dep.exists()
 }
+
+
+// For building exoplayer-specific sample apps, which are located under the 'exoplayer' directory.
```

Comment:
For now it's just a mechanism to filter out sample applications by player. So this creates an exoplayer directory where we can dump sample applications and an README.